workflow/scripts/constants.py

```diff
@@ -463,280 +463,112 @@
 If you want to use the default ATB technology, the minimum must be defined: pypsa-name, technology, crp.
 If a default classificiation is not defined by ATB, then the remaining fields will be used to extract data from the ATB.
 pypsa-name:{
-    "technology":"ATB-Name",
-    "name":"ATB-tech-abreviation",
-    "alias":"ATB-tech-alias.",
-    "detail":"ATB-tech-detail",
-    "crp":"ATB-crp",
+    "display_name": "Utility-Scale Battery Storage - 4Hr",
```

Comment:
for consistency with rest of workflow, should `display_name` be `nice_name`? Or is there a reason for the difference?